<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameForm</class>
 <widget class="QWidget" name="NameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hamsters Name Search</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Name_Button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Names</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_Button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qwidget</class>
   <extends>QWidget</extends>
   <header>qwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
